ign_ros2_control/src/ign_ros2_control_plugin.cpp

```diff
@@ -325,14 +326,17 @@ void IgnitionROS2ControlPlugin::Configure(
     argv.push_back(reinterpret_cast<const char *>(arg.data()));
   }
 
+  // Create a default context, if not already
   if (!rclcpp::ok()) {
     rclcpp::init(static_cast<int>(argv.size()), argv.data());
-    std::string node_name = "ignition_ros_control";
-    if (!controllerManagerPrefixNodeName.empty()) {
-      node_name = controllerManagerPrefixNodeName + "_" + node_name;
-    }
-    this->dataPtr->node_ = rclcpp::Node::make_shared(node_name);
   }
+
+  std::string node_name = "ignition_ros_control";
+  if (!controllerManagerPrefixNodeName.empty()) {
+    node_name = controllerManagerPrefixNodeName + "_" + node_name;
+  }
```

Comment:
I am not really sure why this here is needed? IMHO we should only work with namespaces. This doesn't look much ROS-y and it is probably just confusing people. @sp-sophia-labs What do you think? Did this added some confusion for you when working on this?

Reply:
this is targeting `master` I'm fine removing it, sorry for the back and forth